<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QDialog" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>740</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>订单预支付界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(251,102,102, 200), stop:1 rgba(20,196,188, 210));</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 50px;
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="10" column="0">
       <spacer name="verticalSpacer_14">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>45</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="21" column="4">
       <widget class="QPushButton" name="sure">
        <property name="styleSheet">
         <string notr="true">QPushButton {
height: 30px;
font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;

}
QPushButton:hover {
    background-color: rgb(22,218,208);
}
QPushButton:pressed {
    background-color: rgb(17,171,164);
}
</string>
        </property>
        <property name="text">
         <string>提交订单</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <spacer name="verticalSpacer_18">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 25 10pt '微软雅黑 Light';</string>
        </property>
        <property name="text">
         <string>座位选择</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1" colspan="4">
       <widget class="QFrame" name="frame_4">
        <property name="styleSheet">
         <string notr="true">QFrame {
font: 25 12pt '微软雅黑 Light';
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:10px;
}

QFrame QLabel {
    border: none; /* 确保QLabel没有边框 */
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QLabel" name="beginTime">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>10</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
/*    background-image: url(':/new/prefix1/image/hezhao.jpg');  */
    background-repeat: no-repeat;
    background-position: center;
}</string>
         </property>
         <property name="text">
          <string>        起飞时间</string>
         </property>
        </widget>
        <widget class="QLabel" name="endTime">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>10</y>
           <width>161</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>        降落时间</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="8" column="1">
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="17" column="1" colspan="4">
       <widget class="QGroupBox" name="groupBox">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:10px;
}

QGroupBox QLabel {
    border: none; /* 确保QLabel没有边框 */
}

QGroupBox:hover {
    background-color: #fafff8;
}</string>
        </property>
        <property name="title">
         <string>支付方式</string>
        </property>
        <widget class="QRadioButton" name="rbtn_wechat">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>71</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>微信</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbtn_zhifubao">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>10</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>支付宝</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbtn_creditcard">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>10</y>
           <width>118</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>银行卡</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_12">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>35</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="9">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="13" column="1">
       <spacer name="verticalSpacer_11">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="1" colspan="4">
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">QFrame {
font: 25 12pt '微软雅黑 Light';
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;

}
/*QFrame:hover {
    background-color: #fafff8;
}*/</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>71</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>A座</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>10</y>
           <width>71</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>B座</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>10</y>
           <width>71</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>C座</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>10</y>
           <width>71</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>D座</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_5">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>10</y>
           <width>71</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>E座</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">QFrame {
font: 25 12pt '微软雅黑 Light';
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:10px;
}

QFrame QLabel {
    border: none; /* 确保QLabel没有边框 */
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QLabel" name="beginLocation">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>10</y>
           <width>69</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>始发地</string>
         </property>
        </widget>
        <widget class="QLabel" name="endLocation">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>10</y>
           <width>69</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>目的地</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">QFrame {
font: 25 14pt '微软雅黑 Light';
background-color: #5ce5e5;
border:2px solid #54d1d1;
border-radius:5px;
}

QFrame QLabel {
    border: none; /* 确保QLabel没有边框 */
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="flight_ID">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>航班编号</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="11" column="4">
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <widget class="QLCDNumber" name="countDown"/>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer_13">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>35</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="12" column="9">
       <spacer name="verticalSpacer_15">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="0">
       <spacer name="verticalSpacer_16">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="15" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="22" column="4">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="20" column="4">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="4">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>570</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="12" column="4">
       <widget class="QPushButton" name="selectPassenger">
        <property name="styleSheet">
         <string notr="true">QPushButton {
height: 30px;
font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;

}
QPushButton:hover {
    background-color: rgb(22,218,208);
}
QPushButton:pressed {
    background-color: rgb(17,171,164);
}
</string>
        </property>
        <property name="text">
         <string>选择乘机人</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="17" column="0">
       <spacer name="verticalSpacer_17">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>45</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="1" colspan="3">
       <widget class="QFrame" name="frame_5">
        <property name="styleSheet">
         <string notr="true">QFrame {
font: 25 12pt '微软雅黑 Light';
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:10px;
}

QFrame QLabel {
    border: none; /* 确保QLabel没有边框 */
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>151</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>价格明细(单人)：</string>
         </property>
        </widget>
        <widget class="QLabel" name="showPrice">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>10</y>
           <width>81</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>价格</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="9" column="2" colspan="3">
       <widget class="QLabel" name="note">
        <property name="styleSheet">
         <string notr="true">font: 290 15pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 85, 127);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
